### PR TITLE
Apply new tag styles in select inputs multivalue

### PR DESCRIPTION
**Pull request by ddouglasz**
#### Summary

Apply new tag styles in select inputs multi value

## Description

Before:

| | isDisabled: false  | isDisabled: true |
|--------------------|--------|-----------------|
|Before |image |image|
|After | image|image|

### CODE REVIEW

**Review by FilPob — APPROVED** [review]

**Comment by FilPob** [comment]
Thanks :)

File: packages/components/inputs/select-utils/src/create-select-styles.ts [hunk 1]
```diff
@@ -449,15 +449,22 @@ const menuPortalStyles = (props: TProps) => (base: TBase) => ({
...
-const multiValueStyles = () => (base: TBase) => {
+const multiValueStyles = (props: TProps) => (base: TBase) => {
   return {
     ...base,
     display: 'flex',
     alignItems: 'center',
     height: '32px',
-    backgroundColor: designTokens.colorNeutral95,
-    padding: '0',
+    backgroundColor: (() => {
```
Comment (CarlosCortizasCT) [inline packages/components/inputs/select-utils/src/create-select-styles.ts:458]: Perhaps we can extract this to a function an place it at the top of the file the same we do for other similar calculations (eg: `renovate/major-all-flopflip-packages`)

Comment (ddouglasz) [reply packages/components/inputs/select-utils/src/create-select-styles.ts:460]: Updated here: https://github.com/commercetools/ui-kit/pull/2780/commits/415007433042f7e6889d00d7384adab592b95ed9

Thank you

Comment (CarlosCortizasCT) [reply packages/components/inputs/select-utils/src/create-select-styles.ts:460]: Thanks for updating the `controlStyles` part, but I think we should also do the same here for the `multiValueStyles`.

Comment (ddouglasz) [reply packages/components/inputs/select-utils/src/create-select-styles.ts]: Oh! missed that.
Updated here: https://github.com/commercetools/ui-kit/pull/2780/commits/50df31e2d57a38dba5859cf4ce6117e352201809

Thank you

**Review by CarlosCortizasCT — APPROVED** [review]
Looks good to me 👍 

Thanks!

**Review by kark — APPROVED** [review]
Thanks, looks great!

**Pull request merged**